### DISCUSSION TITLE
How to force Gluetun disconnect/reconnect to re-establish VPN Tunnel

### DISCUSSION THREAD

**Discussion by WA2FAST** [qdm12/gluetun#2142] [Q&A]
Using Gluetun in a docker-compose stack... is there a way to force GlueTun to disconnect from the connected VPN (OpenVPN connection) and re-connect without killing the whole stack and bringing it back up?  I've searched and cannot find an answer to this, sorry if it has already been covered, thank you for your help in advance.

Using an OpenVPN connection on a straightforward docker-compose stack on a Synology NAS.

**Comment by bluskye** [comment]
Not sure how Synology manages containers, but in Portainer you simply restart the container.  The stack is only to create a container with options.

**Reply by WA2FAST** [reply]
Thanks bluskye, I actually have portainer installed however didn't create the stack IN portainer so although I have limited visibility of the stack itself I have never tried to do anything as it's always complaining that the stack wasn't created in portainer.

After looking at this closer in Portainer, I see that I do have the ability/option to restart the container in the stack, so next time I'll give that a shot.  Perhaps I'll try building the stack within portainer too just to give that a shot as well.

Thanks for your help!
